docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -2,16 +2,24 @@
 
 = Deploying a {Ceph} ingress daemon
 
-To match the cardinality diagram, you use cephadm labels to refer to a group of nodes where a given daemon type should be deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
-`HAProxy` is managed by {OpenStackPreviousInstaller} through `Pacemaker`: the three running instances at this point will point to the old RGW backends, resulting in a broken configuration.
-ifeval::["{build}" != "upstream"]
-Since you are going to deploy the Ceph ingress daemon, the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
-endif::[]
-ifeval::["{build}" != "downstream"]
-Since you are going to deploy the https://github.com/openstack-archive/tripleo-ansible/blob/stable/wallaby/tripleo_ansible/ansible_plugins/modules/ceph_mkspec.py[Ceph ingress daemon], the first thing to do is remove the existing `ceph_rgw` config, clean up the config created by {OpenStackPreviousInstaller} and restart the service to make sure other services are not affected by this change.
-endif::[]
-After you complete this procedure, you can reach the RGW backend from the ingress daemon and use RGW through the {object_storage} command line interface (CLI).
-//kgilliga: I will rewrite this intro for GA.
+To deploy the Ceph ingress daemon, you perform the following actions:
+
+. Remove the existing `ceph_rgw` configuration.
+. Clean up the configuration created by {OpenStackPreviousInstaller}.
+. Restart the service to ensure that other services that consume the {object_storage_first_ref} are not affected by this change.
+
+When you deploy the ingress daemon, two new containers are created:
+
+* HAProxy, which you use to reach the back ends.
+* Keepalived, which you use to own the virtual IP address.
+
+To ensure that you distribute the correct number of services to your available {Ceph} nodes, you use `cephadm` labels to refer to a group of nodes where a given daemon type is deployed. For more information about the cardinality diagram, see xref:ceph-daemon-cardinality_migrating-ceph[{Ceph} daemon cardinality].
```

Comment:
Can we update L16 to also say that we're going to use the `rgw` label (which is mentioned later in the procedure  (around L100). This ensure that the distribution of the ingress daemon is reduced to the amount of nodes destined to host RGW daemons (typically 3, so by default 2 of them are selected).